spec/std/signal_spec.cr

```diff
@@ -19,44 +19,48 @@ pending_interpreted describe: "Signal" do
   end
 
   {% unless flag?(:win32) %}
+    # can't use SIGUSR1/SIGUSR2 on FreeBSD because Boehm uses them to suspend/resume threads
+    signal1 = {% if flag?(:freebsd) %} Signal.new(LibC::SIGRTMAX - 1) {% else %} Signal::USR1 {% end %}
+    signal2 = {% if flag?(:freebsd) %} Signal.new(LibC::SIGRTMAX - 2) {% else %} Signal::USR2 {% end %}
```

Comment:
**question:** Could we use RT signals on all platforms, instead of `USR1`, `USR2`?

Reply:
Maybe, though I'm not sure all platforms have SIGRT signals, and USR1 and USR2 should usually be available ... I wonder why would Boehm use USR1 and USR2 when it can use SIGRT signals (and does so, but only when told to) 🤷